packages/core/src/dispatcher.ts

```diff
@@ -310,10 +310,14 @@ export class Dispatcher {
   async _handleUpdateMessage(message: UpdateMessage): Promise<void> {
     // TODO Add validation the message adheres to the proper format.
 
-    const { stream: streamId, tip } = message
+    /*
+      @active-branch
+    */
+    console.log("_handleUpdateMessage")
```

Comment:
don't forget to clean up extra logs used during development before sending code out for review